docs/doxygen/images/retry_utils_flow.pu

```diff
@@ -5,10 +5,12 @@ skinparam ArrowFontSize 18
 
 start
 : **RetryUtils_ParamsReset()**;
+note: Initializes parameters
 repeat
-	: Server communication attempt;
+	: Attempt communication with Server;
 	if (Success) then (no)
     	: **RetryUtils_BackoffAndSleep()**;
+		note: Performs backoff calculation and delay
```

Comment:
This note more than doubled the width of the image, but it still takes up 25% of the viewport width.